modules/guides/examples/connect/connect-cbc.sh

```diff
@@ -1,20 +1,22 @@
 #!/usr/bin/env sh
 
+# Will need to rewritten for Elixir
+
 # tag::cbc-connect[]
-cbc ping -u Administrator -P password -U couchbase://localhost/travel-sample \
+cbc ping -u Administrator -P password -U couchbase://localhost/$DATABASE_NAME \
 	--count=1 \
 	--table
 # end::cbc-connect[]
 
 # tag::cbc-connect-cert[]
-cbc ping -v -U "couchbases://127.0.0.1/travel-sample?certpath=ca.pem" \
+cbc ping -v -U "couchbases://127.0.0.1/$DATABASE_NAME?certpath=ca.pem" \
 	--count=1 \
 	--table
 # end::cbc-connect-cert[]
 
 # tag::cbc-connect-capella[]
 cbc ping -u username -P "passworD#1" \
-	-U "couchbases://cb.oawlpi4audpc6jp5.cloud.couchbase.com/travel-sample?certpath=cert.pem" \
+	-U "couchbases://cb.oawlpi4audpc6jp5.cloud.couchbase.com/$DATABASE_NAME?certpath=cert.pem" \
```

Comment:
Should the database name use curly braces to delimit the end of the env variable? (Thinking about automated example file testing)

```suggestion
	-U "couchbases://cb.oawlpi4audpc6jp5.cloud.couchbase.com/${DATABASE_NAME}?certpath=cert.pem" \
```

Reply:
using <BUCKET_NAME>